go/vt/vttablet/tabletserver/tx_prep_pool.go

```diff
@@ -194,3 +195,17 @@ func (pp *TxPreparedPool) IsEmpty(tableName string) bool {
 	}
 	return true
 }
+
+// IsEmpty returns true if the pool is empty.
+func (pp *TxPreparedPool) IsEmpty() bool {
+	pp.mu.Lock()
+	defer pp.mu.Unlock()
+	if !pp.twoPCEnabled {
+		return true
+	}
```

Comment:
Just checking, but the bool being false means that any previous work was completed from when it was enabled?

Reply:
Yes exactly. If the two pc is disabled then we don't accept in prepared transaction or commit transaction requests. So there is no progress that is going to be made.